tests/functional/disaster-recovery/regional-dr/test_rdr_bug_verification.py

```
@@ -470,8 +465,9 @@ def test_mco_updates_peer_secret_on_mon_failover(
         )
         logger.info("CephBlockPool is in Ready status")
 
-        # --- Step 10: Verify the updated token exists in the secret created by
-        #              token-exchange-agent on the peer cluster ---
+        logger.test_step(
+            "Verify the updated token is propagated to the " "peer cluster by MCO"
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)